### DISCUSSION TITLE
为什么不采用，起诉退房，合法停贷的方法

### DISCUSSION THREAD

**Discussion by pirwuwu55** [WeNeedHome/SummaryOfLoanSuspension#1172] [General]
这个方法停贷不会影响征信，那为什么没形成规模呢？

**Comment by Cegazhos** [comment]
起诉解除合同是个人发起的，时间长判决慢。
而集体停贷，共进退有力量，目前也是合法的

**Comment by xijinping0** [comment]
“你说法治我都觉得好笑！”

<a href="https://youtu.be/6paeOLgiGFc?t=103s">
  <img width="993" alt="Screen Shot 2022-09-07 at 21 08 29 PM" src="https://user-images.githubusercontent.com/111333464/189032331-78c72896-9d74-484f-8de4-2b7cab4aa576.png">
</a>

**Comment by oceanzhang860411** [comment]
你國有法治？你說吧，哪個法你覺得有效實施了的

**Reply by xijinping0** [reply]
中国有效实施的法有：领导的看法、领导的想法、领导的说法！

**Reply by oceanzhang860411** [reply]
這也太量子力學了，領導的想法也許今天和明天不一樣

**Comment by xiejun946** [comment]
我试过，因为合法途径，需要成本和时间，半年到一年，律师费1.5万到2万之间。而且首付一定没了。是一定。我当时准备起诉的时候，开发商还没失信，过了半年就失信了不还了。

**Reply by xiejun946** [reply]
而且房子现在开发商拿去给别人置换或者重新卖了

**Reply by yyanyi213** [reply]
我目前载进去的钱差不多是20万，我也想退房，放弃首付。请问你起诉的时候，律师怎么确实收费金额的？自己来起诉行不行，因为我看购房合同明确说了违约6个月可以退房。
请问你现在房贷也取消了吧？
